app/src/main-process/main.ts

```diff
@@ -188,6 +188,18 @@ app.on('ready', () => {
     }
   )
 
+  function setAllMenuItems(enabled: boolean) {
```

Comment:
Probably the part that makes me most uncomfortable about this change is that we might re-enable the whole menu when the modal is dismissed. After more testing on macOS I see this re-enables all menu items when you have no repositories selected (including features you shouldn't have access to).

I'll see if this is easy to trigger or if it requires tracking the current menu state _somehow_...